app/models/comment.rb

```diff
@@ -2,9 +2,9 @@
 
 class Comment < ApplicationRecord
   belongs_to :post
-  belongs_to :user
+  belongs_to :author
 
   def increment_comments
-    user.increment!(:PostsCounter)
+    post.increment!(:PostsCounter)
```

Comment:
- [ ] As highlighted by the previous reviewer we should increase the `:CommentsCounter`. The post table does not have a `PostsCounter` attribute.

  ```
  def increment_comments
    post.increment!(:CommentsCounter)
  end
  ```